bootstrap.sh

```diff
@@ -7,7 +7,14 @@ pip3 install --upgrade pip
 
 # Installing pipenv, create new env and sync lib with pipfile
 pip3 install pipenv
-cd /vagrant/givitsite/
+
+#navigate to app folder
+if [ `logname` = "vagrant" ]; then
```

Comment:
This is one option to find out in which environment are you.
Another option is to look for the CI environment variable. This env var is set by GitHub Action, while it is not set by Vagrant.